bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -56,6 +56,51 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
             docker_image='gcr.io/moz-fx-data-airflow-prod-88e0/bigquery-etl:latest',
             owner='{{ task.owner }}',
             email={{ task.email | sort }},
+    {% elif task.is_dq_check -%}
+       {{ task.task_name }} = bigquery_dq_check(
+            task_id='{{ task.task_name }}',
+            {#+ TODO when Airflow is updated to 2.2+ use ds_nodash instead of ds_format -#}
+            source_table={%+ if task.date_partition_offset -%}'{{ task.destination_table }}${% raw %}{{{% endraw %} macros.ds_format(macros.ds_add(ds, {{ task.date_partition_offset }}), "%Y-%m-%d", "%Y%m%d") {% raw %}}}{% endraw %}'
+                            {%+ elif task.destination_table -%}'{{ task.destination_table }}'
+                            {%+ else -%}None
+                            {%+ endif -%},
+            {%+ if task.query_project -%}
+            dataset_id='{{ task.project }}:{{ task.dataset }}',
+            project_id='{{ task.query_project }}',
+            {%+ if not task.query_file_path -%}
+            sql_file_path="sql/{{task.project}}/{{task.dataset}}/{{task.destination_table}}/checks.sql",
```

Comment:
We can remove `sql_file_path` from the template to have one less parameter in the Task.  `sql_file_path` will default to `{project_id}/{dataset_id}/{source_table}`.  If the parameter is not removed, we can remove the `checks.sql` part.  Having the filename implies that a different filename can be used however the current implementation will only use `checks.sql`.